### ISSUE TITLE
Welcome to Chef Automate should only be shown once

### ISSUE THREAD

**Issue by tas50** [chef/automate#4901]
## Describe the bug
Right now you see the `Welcome to Chef Automate` each time you login to a new browser or remove all your cookies. This is not only annoying, but it results in more users opting out of telemetry. We need to be storing the completion of this data on the service side not showing this every time they enter via a new browser.

## To Reproduce
- Create new user
- Login and close the welcome message, but do not select "show this message at launch"
- Login via new browser and notice you see the message again

## Expected behavior
You should only be welcomed once.

## Versions (please complete the following information):
- OS: macOS
- Browser: Firefox and Chrome
- Automate Build Number: 20210324172345

**Comment by kalroy** [comment]
@jong-progress What's your thought on this?

**Comment by jong-progress** [comment]
@kalroy yes. I agree. We should only it show it once.  But it would be cool if they had access to it via an About Dialogue from the User Drop down.  
So that  firs time users have access to those documentation links, Learn Chef Links, etc. to learn more about Chef.
